src/planscape/conditions/migrations/0004_get_condition_pixels.py

```diff
@@ -0,0 +1,104 @@
+from django.db import migrations
+from typing import Tuple
+
+# Get condition pixels fetches raw raster values for all pixels that intersect
+# with a given geometry.
+# Inputs include 1) raster details:
+#   - table name (table, schema),
+#   - raster name (raster_name)
+#   - and relevant raster fields (raster_name_column, raster_column)
+# and 2) geometry
+#   - a shape in EWKB format
+#
+# Along with raw values, data indicating pixel position is returned. Table
+# outputs include ...
+#   - the top-left coordinate (xcoord, ycoord)
+#   - pixel position relative to the top-left coordinate (x, y)
+#   - raw raster pixel value (value)
+#
+# An example call from Django python may be ...
+# geo = Polygon(...)
+# with connection.cursor() as cursor:
+#     cursor.callproc(
+#                'get_condition_pixels',
+#                ('conditions_conditionraster', 'public', 'biodiversity',
+#                 'name', 'raster', geo.ewkb))
+SQL = """
+create or replace function get_condition_pixels(
+      param_table text,
+      param_schema text,
+      param_raster_name text,
+      param_raster_name_column text,
+      param_raster_column text,
+      param_geom_ewkb bytea) returns TABLE(
+                                         xcoord float,
```

Comment:
maybe to clarify what the values represent we could use the following names?
x_origin
y_origin
x_dist
y_dist

Reply:
"origin" might be confusing because it implies (0,0) and positive pixels are upwards and/or to the right. postGIS calls it upperLeft - maybe I'll call them: upper_left_coord_x, upper_left_coord_y.

"dist" might be confusing because it implies some unit. I can call it pixel_dist_x, pixel_dist_y.